packages/rpc-graphql/src/schema/account.ts

```diff
@@ -19,7 +19,7 @@ export const accountTypeDefs = /* GraphQL */ `
         encoding: String
         executable: Boolean
         lamports: BigInt
-        owner: Account
+        ownerProgram: Account
         rentEpoch: BigInt
```

Comment:
<i class='graphite__hidden'>[Re: lines 18 to 23]<br/></i>

This is where I imagine `space` belongs. No?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1837?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
It probably does now, yeah. But I think before 1.16 it wasn't so, right?